guardrails/guard.py

```diff
@@ -359,6 +376,20 @@ def __call(
             if prompt_params is None:
                 prompt_params = {}
 
+            # TODO: Add tracing: guard usage and user usage for Validator Hub
+            # Start a span for this guard call
+            with self._hub_tracer.start_as_current_span("/guard_call") as span:
+                # Inject the current context
+                self._hub_telemetry.inject_current_context()
+
+                span.set_attribute("guard_id", self._guard_id)
+                span.set_attribute("user_id", self._user_id)
+                span.set_attribute("llm_api", llm_api.__name__ if llm_api else "None")
+                span.set_attribute("custom_reask_prompt", self.reask_prompt is not None)
+                span.set_attribute(
+                    "custom_reask_instructions", self.reask_instructions is not None
+                )
```

Comment:
Since this isn't capturing anything except custom attributes, can this be moved to a util method somewhere?

Reply:
Added a util method here: https://github.com/guardrails-ai/guardrails/blob/5e4096be30ceb4df056afbeb6458b214c8e926ad/guardrails/utils/hub_telemetry_utils.py#L89